### ISSUE TITLE
[BITV] 9.2.4.3/3.6 - Modal dialog "Choose target folder" - When the dialog is open, it is still possible to navigate through the elements in the background using keyboard, which should be prevented. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36992]
- [x] https://github.com/nextcloud/server/pull/37780

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v3-n6

</details>

- [x] 9.2.4.3/10.6 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v10-n6
